.env

```diff
@@ -1,21 +1,14 @@
-# Environment variables
-WORKSPACE_ID=
-CONVERSATION_USERNAME=
-CONVERSATION_PASSWORD=
-#Optional cloudant URL for logging
-CLOUDANT_URL=
-#If cloudant url is specified a user name and password must be specified to secure the logging endpoints
-LOG_USER=
-LOG_PASS=
-NLU_USERNAME=
-NLU_PASSWORD=
-
-TONE_ANALYZER_USERNAME=
-TONE_ANALYZER_PASSWORD=
-
-RNR_PASSWORD=

```

Comment:
great to see the R&R stuff go away as that is the direction we hear from the watson product team is that we should use discovery from now on.